doc/contribute/build_ccf.rst

```diff
@@ -70,17 +70,36 @@ Although CCF's unit tests can be run through ``ctest`` directly, the end-to-end
 
     Tests that require enclave attestation will be skipped.
 
-Updating the Documentation
---------------------------
+Build Older Versions of CCF
+---------------------------
+
+Building older versions of CCF may require a different toolchain than the one used to build the current ``main`` branch (e.g. 1.x CCF releases are built with `clang-8`). To build an old version of CCF locally without having to install another toolchain that may conflict with the current one, it is recommended to use the ``ccfciteam/ccf-ci`` docker image. The version tag of the ``cci-ci`` image used to build the old version can be found in the :ccf_repo:`.azure-pipelines.yml` YAML file (under ``resources:container:image``).
+
+.. code-block:: bash
+
+    $ export CCF_CI_IMAGE_TAG="oe0.17.2-clang-8" # e.g. building CCF 1.0.15
+    $ export LOCAL_CCF_CHECKOUT_PATH=/path/to/local/ccf/checkout
+    $ cd $LOCAL_CCF_CHECKOUT_PATH
+    $ git checkout ccf-1.0.15 # e.g. building CCF 1.0.15
+    $ docker run -ti --device /dev/sgx_enclave:/dev/sgx_enclave --device /dev/sgx_provision:/dev/sgx_provision -v $LOCAL_CCF_CHECKOUT_PATH:/CCF ccfciteam/ccf-ci:$CCF_CI_IMAGE_TAG bash
+    # container started, following lines are in container
+    $ root@:/# cd CCF/
+    $ root@:/# mkdir build_docker && cd build_docker
+    $ root@:/# cmake -GNinja .. && ninja
```

Comment:
I know these are probably copied verbatim, but it's confusing to mix use of `#` as comment delimiter and command prompt. It's highlighted as a comment in this GitHub diff, and probably will be within the rendered code-block.

```suggestion
    # container started, following lines are in container
     $ cd CCF/
     $ mkdir build_docker && cd build_docker
     $ cmake -GNinja .. && ninja
```

Reply:
Yeah, I wasn't sure what would be clearer and went for the `/#` because it didn't actually render as a comment in Sphinx. I think the "container started..." comment above is sufficient so the `$` is clearer you're right.